components/TextAreaDetails.tsx

```diff
@@ -3,13 +3,20 @@ import {
   DynamicComponentControlledProps,
   DynamicComponent
 } from "remultiform/component-wrapper";
+import {
+  Checkboxes as LBHCheckboxes,
+  CheckboxItem
+} from "lbh-frontend-react/components";
 
 import PropTypes from "../helpers/PropTypes";
 
 import { Details } from "./Details";
 import { TextArea } from "./TextArea";
 
-type Props = DynamicComponentControlledProps<string> & {
+type Props = DynamicComponentControlledProps<{
+  value: string;
+  isPostVisitAction?: boolean;
```

Comment:
I think this should be required, and default to `false` when we're not showing a checkbox.

Reply:
This would make the types in the schema required, too.